tests/utils/rpc/dev.go

```diff
@@ -0,0 +1,76 @@
+// Copyright 2021 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package rpc
+
+import (
+	"context"
+	"encoding/binary"
+	"fmt"
+	"time"
+
+	"github.com/ChainSafe/gossamer/lib/common"
+)
+
+// PauseBABE calls the endpoint dev_control with the params ["babe", "stop"]
+func PauseBABE(ctx context.Context, rpcPort string) error {
+	endpoint := NewEndpoint(rpcPort)
+	const method = "dev_control"
+	const params = `["babe", "stop"]`
+	_, err := Post(ctx, endpoint, method, params)
```

Comment:
@EclesioMeloJunior babe can be controlled by RPC?

Reply:
@timwu20 yes! In the `./dot/rpc/modules/dev.go` we expose the method `Control` which calls the `blockProducerAPI.Pause()/Resume()`